common/changes/@uifabric/charting/PrograssBarChart_2018-07-24-09-13.json

```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "@uifabric/charting",
-      "comment": "Adding Stacked Bar chart",
+      "comment": "Change import path for package",
```

Comment:
FWIW this comment here gets put in release notes for the package. And "change import path for package" isn't a great release note.

We should make the `npm run change` more clear on that, but you want to make sure the language in this makes sense for that.

You can see a release note example here:
https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/%40uifabric%2Fcharting_v0.11.0